### DISCUSSION TITLE
Fetch changelogs from NPM Code for public, but not open-sourced packages

### DISCUSSION THREAD

**Discussion by khitrenovich** [renovatebot/renovate#28007] [Suggest an Idea]
### Tell us more.

Some packages are public, but not fully open-sourced - meaning that they are available to fetch from NPM, but their source is not open to public. A good example is `@stytch/react` - their `CHANGELOG.md` is available on https://www.npmjs.com/package/@stytch/react?activeTab=code and can be seen in the [code diff](https://app.renovatebot.com/package-diff?name=@stytch%2freact&from=15.0.0&to=16.0.0#d2h-211439), but does not appear in Renovate PRs. It would be great to have it fetched!

**Comment by rarkins** [comment]
Duplicate of https://github.com/renovatebot/renovate/discussions/24778
